glib/src/gobject/flags.rs

```diff
@@ -3,6 +3,7 @@
 use crate::translate::*;
 
 bitflags::bitflags! {
+    #[doc(alias = "GParamFlags")]
     pub struct ParamFlags: u32 {
         const READABLE = 1;
```

Comment:
Would be nice if those are using the ffi values. Also they seem to lack a doc alias

Reply:
Ah indeed! One stone, two pigeons. :D (I need the first to get the second)